values.yaml

```diff
@@ -32,6 +32,16 @@ global:
   #   image: "hashicorp/consul-enterprise:1.5.0-ent"
   image: "consul:1.7.1"
 
+  # array of image pull secret names that will be applied to each service
+  # account. This can be used to reference image pull secrets if using
+  # a custom consul or consul-k8s Docker image.
+  # See https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry.
+  #
+  # Example:
+  #   imagePullSecrets:
+  #   - name: my-secret-name
```

Comment:
These example feels confusing. When empty, the value needs `[]`, but it doesn't need them when I give a value? In the [`extraConfig`](https://github.com/hashicorp/consul-helm/blob/master/values.yaml#L237) text we mention that it's an array of objects, which might help a bit.

Reply:
Yeah good call, changed to say "array of objects containing image pull secret names"